users/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const cors = require('cors');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ce-box/CE5508-astraoke-api/pullRequest?prid=8478842)
```suggestion
const cors = require("cors");
```